### ISSUE TITLE
Success and Warning buttons need appropriate color contrast with active/focus

### ISSUE THREAD

**Issue by joshuasosa** [az-digital/arizona-bootstrap#609]
## Problem/Motivation
Text on backgrounds should have sufficient color contrast to meet WCAG AA for normal text.

### Describe the bug
In the currently published Bootstrap version (2.0?), the buttons for Success and Warning types get the wrong text color when their state is 'active' or 'focus', such as when clicking on them.

https://digital.arizona.edu/arizona-bootstrap/docs/2.0/components/buttons/

### To Reproduce
Click on the Success or Warning buttons to see the color change as the button is clicked. Or use inspector tools to manually set Success or Warning button status to active or focus.

## Proposed resolution
Description of the proposed solution, the rationale behind it, and workarounds for people who cannot use the patch.

### Expected behavior
Success buttons for Success and Warning should retain their respective 'hover' state text color.

### Success and Warning are 'active'
![image](https://user-images.githubusercontent.com/4400438/202592897-4ed038ce-149f-46d5-956d-6596e7d87bb3.png)

### Colors fail for normal text
https://webaim.org/resources/contrastchecker/
![image](https://user-images.githubusercontent.com/4400438/202593110-027e981a-2677-4d9c-ba76-43a6f2e8afe8.png)
![image](https://user-images.githubusercontent.com/4400438/202593145-ade1cbf0-d253-488e-a03b-ae63cc1e2f49.png)

**Comment by joeparsons** [comment]
Possibly related issue: #244
